src/pages/ProjectsListPage.js

```diff
@@ -14,11 +14,15 @@ const ProjectsList = ({ projects, loading }) => {
 
   return (
     <ul>
-      {projects.map(project => (
-        <li key={project.id}>
-          <Link to={`/projects/${project.id}`}>{project.name}</Link>
-        </li>
-      ))}
+      {projects
+        .filter(project => project.status === 'Ideation')
+        .map(project => (
+          <li key={project.id}>
+            <Link to={`/projects/${project.id}`}>
+              {project.description || project.name}
```

Comment:
It seems like it should be the project name, followed by the project description to me